src/components/paragraph/rows/three-column.tsx

```diff
@@ -44,7 +44,12 @@ const ThreeColumn = async ({items, fullWidth = true, config}: Props) => {
       })}
     >
       <div
-        className="centered flex flex-col justify-between gap-90 *:mx-auto *:w-full md:flex-row md:flex-wrap *:md:w-[calc(50%_-_5rem)] lg:flex-nowrap *:lg:w-[calc(33.3%_-_5rem)]"
+        className={clsx(
+          "centered flex w-full flex-col justify-between gap-90 *:mx-auto *:w-full md:flex-row md:flex-wrap *:md:w-[calc(50%_-_5rem)] lg:flex-nowrap *:lg:w-[calc(33.3%_-_5rem)]",
+          {
+            "md:w-[124rem] lg:w-[124rem]": fullWidth,
```

Comment:
Would it work to just set max width and remove the centered class?

Reply:
Ooo so I gave that a try -- we'll want to keep it as width instead of max-width and keep `centered` class to ensure that we still have the side spacing to prevent the cards from going edge to edge 👀 

Example: `width`
<img width="820" height="1293" alt="image" src="https://github.com/user-attachments/assets/8c995e03-6208-463e-96ff-d09ff281c790" />


Example: `max-width`
<img width="820" height="1293" alt="image" src="https://github.com/user-attachments/assets/33918ee7-ec09-4592-b0c3-aee7818eaae9" />

Reply:
I'm so sorry, @pookmish 🤦‍♀️ I'm an absolute fool.
> Looks like it's missing the gutters like we have in the other projects

I see exactly what you mean now https://github.com/SU-SWS/sulgryphon-nextjs/blob/1.x/src/styles/tailwind/plugins/centered-container.tsx

I'll retest this morning and also get those additional fixes in!